designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/BaseEndpointDefinitions.scala

```diff
@@ -84,6 +84,16 @@ object BaseEndpointDefinitions {
 
   }
 
+  def toTextPlainCodecSerializationOnly[T](
+      toMessage: T => String
+  ): Codec[String, T, CodecFormat.TextPlain] =
+    Codec.string.map(
+      Mapping.from[String, T](deserializationNotSupportedException)(toMessage)
+    )
+
+  private def deserializationNotSupportedException =
+    (_: Any) => throw new IllegalStateException("Deserializing errors is not supported.")
```

Comment:
when you search for `throw new IllegalStateException("Des` you will find we have sth similar in several places. Could you please refactor the places? It looks like sth simple to do

Reply:
LOL, it has spread a lot, I only search by method name.